src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -611,6 +620,22 @@ public static void GetFullPath_Windows_ValidExtendedPaths(string path)
             }
         }
 
+        [PlatformSpecific(TestPlatforms.Windows)]  // Tests Windows-specific paths
+        [Theory]
+        [InlineData(@"\\.\UNC\")]
+        [InlineData(@"\\.\UNC\server7")]
```

Comment:
I made them server7 etc because it's easier to figure out which test a failure corresponds to. In xunit/Helix they seem to pick up extra slashes. I only did it for test cases that would not hit the disk.